scripts/deploy-perp-only-market.s.sol

```diff
@@ -21,12 +21,23 @@ import "./config-mainnet.sol";
 
 
 /**
- * MARKET_NAME=weth forge script scripts/deploy-market.s.sol --private-key {} --rpc {} --broadcast
+ * IS_MAINNET=False MARKET_NAME=AAVE PRIVATE_KEY={} forge script scripts/deploy-perp-only-market.s.sol --private-key {} --rpc-url {} --verify --verifier blockscout--verifier-url {} --broadcast --priority-gas-price 1
  **/
+
+// MAINNET 
+// RPC: https://rpc.lyra.finance
+// VERIFIER: https://explorer-prod-testnet-0eakp60405.t.conduit.xyz/api
+
+// TESTNET
+// RPC: https://rpc-prod-testnet-0eakp60405.t.conduit.xyz
+// VERIFIER: https://explorer.derive.xyz/api
+
+// will need to use an API key endpoint as limits will get hit
 contract DeployPerpOnlyMarket is Utils {
 
   /// @dev main function
   function run() external {
+    bool isMainnet = vm.envBool("IS_MAINNET");
```

Comment:
I personally like to read the chain id since I flub these ENVs a lot lol. This is fine though

Reply:
good point, done